static/app/components/profiling/profileHeader.tsx

```diff
@@ -49,6 +69,13 @@ function ProfileHeader() {
           ]}
         />
       </Layout.HeaderContent>
+      <Layout.HeaderActions>
+        {transactionTarget && (
+          <Button onClick={handleGoToTransaction} to={transactionTarget}>
```

Comment:
@Zylphrex we also have the transaction in the details drawer - we should make sure we add it there too. Also, any chance we can make this an actual link vs a button and get some semantics html points :D?

Reply:
Yeah let's add it to the details draw too.

This button is an anchor under the hood right now, is that what you mean?